components/breadcrumbs/README.md

```diff
@@ -2,8 +2,68 @@
 
 The `d2l-breadcrumbs` element can be used to help users understand where they are within an application, and provide useful clues about how the space is organized. They also provide a convenient navigation mechanism.
 
+<!-- docs: start hidden content -->
 ![screenshot of d2l-breadcrumbs component](./screenshots/basic.png)
+<!-- docs: end hidden content -->
 
+<!-- docs: demo -->
+```html
+<script type="module">
+  import '@brightspace-ui/core/components/breadcrumbs/breadcrumb.js';
+  import '@brightspace-ui/core/components/breadcrumbs/breadcrumb-current-page.js';
+  import '@brightspace-ui/core/components/breadcrumbs/breadcrumbs.js';
+</script>
+<d2l-breadcrumbs>
+  <d2l-breadcrumb text="Item 1" href="page1.html"></d2l-breadcrumb>
+  <d2l-breadcrumb text="Item 2" href="page2.html"></d2l-breadcrumb>
+  <d2l-breadcrumb-current-page text="Current Page"></d2l-breadcrumb-current-page>
+</d2l-breadcrumbs>
+```
+
+## Best Practices
+<!-- docs: start best practices -->
+<!-- docs: start dos -->
+* Breadcrumbs should be used to show the information architecture of a page, not the history
+* Breadcrumbs are page level controls – only one should appear on a page a time. If a page that normally has breadcrumbs is embedded in another page, remove the breadcrumbs from the embedded page.
+<!-- docs: end dos -->
+
+<!-- docs: start donts -->
+* Do not use breadcrumbs for multi-step processes, use a Stepper (TBD)
+* If your navigation is only one level deep, and never gets deeper than one level, consider other design patterns or components
+<!-- docs: end donts -->
+<!-- docs: end best practices -->
+
+## Responsive Behavior
+
+There are various options to truncate the breadcrumb when in horizontally constrained spaces in order to keep the breadcrumb on one line.
+
+### Limited Width
+
+![screenshot of d2l-breadcrumbs component limited width](./screenshots/limited-width.png)
```

Comment:
Do we want to show these screenshots initially? Seems alright to leave it like this for now but I'd guess a lot of the screenshots/copy text will change when Jeff starts going through the pages

Reply:
Good point. This one might be useful as a demo with code actually, and we can probably hide the compact mode one.